RELEASE.md

```diff
@@ -40,15 +40,27 @@ Please make sure you have these tools installed before starting this process:
 
     This will update the `requirements.lock` with the latest versions and freeze the `worker-manager.config.codacy.worker.image` version on `./codacy/values.yaml`.
 
--   [ ] 6.  Commit
+-   [ ] 6.  Generate release notes
```

Comment:
It's great that we're adding the generation of release notes to the release process of the charts! 🙌

However, we should only merge these new instructions once the teams are actively filling in the release notes in Jira, so that at this point the process is smooth - otherwise all release notes will be empty and it's a lot of work to write them when doing the release.

I have an issue on my side to make this work, that I should be tackling very soon:

https://codacy.atlassian.net/browse/DOCS-58

Reply:
So we should only actually get this PR going after that's done i'd say. :)

Reply:
Yes, I agree.